apps/condo/domains/subscription/schema/SubscriptionContext.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Generated by `createschema subscription.SubscriptionContext 'organization:Relationship:Organization:CASCADE; subscriptionPlan:Relationship:SubscriptionPlan:PROTECT; startAt:DateTimeUtc; endAt:DateTimeUtc; basePrice:Decimal; calculatedPrice:Decimal; appliedRules:Json; isTrial:Checkbox;'`
+ */
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const { MONEY_AMOUNT_FIELD } = require('@condo/domains/common/schema/fields')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const access = require('@condo/domains/subscription/access/SubscriptionContext')
+
+const ERRORS = {
+    END_DATE_MUST_BE_AFTER_START_DATE: {
+        code: BAD_USER_INPUT,
+        type: 'END_DATE_MUST_BE_AFTER_START_DATE',
+        message: 'endAt must be after startAt',
+    },
+}
+
+
+const SubscriptionContext = new GQLListSchema('SubscriptionContext', {
+    schemaDoc: 'Subscription context linking organization to a subscription plan with calculated pricing',
```

Comment:
best to add that in case of subscription plan change you should create new Subscription Context, instead of editing existing one (judging by the update:false)